code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -377,6 +386,14 @@
 							feedback_add_details("wizard_spell_learned","WM") //please do not change the abbreviation to keep data processing consistent. Add a unique id to any new spells
 							add_spell(new/spell/aoe_turf/conjure/pontiac,H)
 							temp = "This spell summons a glorious, flaming chariot that can move in space and through walls.  It also has an extremely long cooldown."
+						if("bundle")
+							feedback_add_details("wizard_spell_learned","SB")
+							new /obj/item/weapon/storage/box/spellbook(get_turf(H))
+							temp = "You have purchased the spellbook bundle."
+							uses -= 4
```

Comment:
It costs all five of your spellpoints to take the bundle, so uses-4 takes all of those spellpoints away. Usually you cannot refund a physical item (which this is) because of max uses. If you purchase a staff of change and 4 spells then refund your spells, you wont get the 1 point back from the staff of change and it wont be refunded. The reason max_uses -=5 is because if it were the same as the other items on the list, you'd be able to purchase the bundle for 5 spellpoints, refund it and receive 4 back, then purchase 4 more spells or whatever. This prevents you from doing that.